test/jdk/javax/swing/JTree/6263446/bug6263446.java

```diff
@@ -38,7 +38,7 @@
 
 public class bug6263446 {
 
-    private static final String FIRST = "AAAAAAAAAAA";
+    private static final String FIRST = "AAAAA";
```

Comment:
any particular reason for decreasing length of string?

Reply:
Yes, as I mentioned to make it same like JTable/6263446/bug6263446.java which has not failed in these OCI systems at all as the cause of this test failure in OCI system was not making any sense but in my local ubutu20.04 system ,it fails once in a while for not able to select the string which is done in the testcase by mouse clicks, so more characters can result in failure in selection if mouse clicks is not getting registered properly...
Reducing characters does not hamper testcase which can still be used to test JDK-6263446 regression